src/backend/opencl/magma/magma_blas_clblas.h

```diff
@@ -0,0 +1,74 @@
+/*******************************************************
+ * Copyright (c) 2014, ArrayFire
+ * All rights reserved.
+ *
+ * This file is distributed under 3-clause BSD license.
+ * The complete license agreement can be obtained at:
+ * http://arrayfire.com/licenses/BSD-3-Clause
+ ********************************************************/
+
+#pragma once
+
+#include <defines.hpp>
+
+#include <clBLAS.h>
+#include <err_clblas.hpp>
+
+// Convert MAGMA constants to clBLAS constants
+clblasOrder          clblas_order_const( magma_order_t order );
+clblasTranspose      clblas_trans_const( magma_trans_t trans );
+clblasUplo           clblas_uplo_const ( magma_uplo_t  uplo  );
+clblasDiag           clblas_diag_const ( magma_diag_t  diag  );
+clblasSide           clblas_side_const ( magma_side_t  side  );
+
+// Error checking
+#define OPENCL_BLAS_CHECK CLBLAS_CHECK
+
+// Transposing
+#define OPENCL_BLAS_TRANS_TYPE clblasTranspose // the type
```

Comment:
Hmm good point. Although it would be safer to use scoped enums, in this case, they would reduce readability. Let's just use unscoped enums for those types for now.